CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.18)
```

Comment:
can you also update the requirements in `install/from_source.rst` to match?

Reply:
Will do, thanks